src/Client/pages/WishList.fs

```diff
@@ -81,7 +82,7 @@ let update (msg:WishListMsg) model : Model*Cmd<WishListMsg> =
     | AddBook ->
         if Validation.verifyBook model.NewBook then
             let wishList = { model.WishList with Books = (model.NewBook :: model.WishList.Books) |> List.sortBy (fun b -> b.Title) }
-            { model with WishList = wishList; NewBook = Book.empty }, postWishListCmd(model.Token,wishList)
```

Comment:
Setting the NewBook back to empty while the fields were still filled caused a usability bug e.g. adding multiple books with the same title.